fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/communication/CellModule.java

```diff
@@ -110,16 +116,17 @@ public Integer sendCam() {
             log.warn("sendCAM: Cell communication disabled (!cellModule.isEnabled()).");
             return null;
         }
-        if (configuration == null) {
+        CellModuleConfiguration.CellCamConfiguration camConfiguration = configuration.getCamConfiguration();
+        if (configuration == null || camConfiguration == null) {
```

Comment:
This has been detected by spotbugs:

You check if `configuration` is null, but this very object is already used in the line above